application/models/Question.php

```diff
@@ -176,10 +182,12 @@ public function rules()
                 $aRules[] = array('title', 'match', 'pattern' => '/comment$/', 'not' => true, 'message' => gT("'comment' suffix can not be used with multiple choice with comments."));
             }
         } else {
-            // Disallow other if sub question have 'other' for title
-            $oSubquestionOther = Question::model()->find("parent_qid=:parent_qid and LOWER(title)='other'", array("parent_qid" => $this->qid));
-            if ($oSubquestionOther) {
-                $aRules[] = array('other', 'compare', 'compareValue' => 'Y', 'operator' => '!=', 'message' => sprintf(gT("'%s' can not be used if the 'Other' option for this question is activated."), 'other'), 'except' => 'archiveimport');
+            if ($this->getAsSubquestions()) {
+                // Disallow other if sub question have 'other' for title
+                $oSubquestionOther = Question::model()->find("parent_qid=:parent_qid and LOWER(title)='other'", array("parent_qid" => $this->qid));
+                if ($oSubquestionOther) {
+                    $aRules[] = array('other', 'compare', 'compareValue' => 'Y', 'operator' => '!=', 'message' => sprintf(gT("'%s' can not be used if the 'Other' option for this question is activated."), 'other'), 'except' => 'archiveimport');
```

Comment:
Can you remind me why the archiveImport is excepted?
Maybe add it as a comment?

Reply:
I think it's for LSA

Reply:
> I think it's for LSA

but unsure … let's grep

Reply:
I would add a comment about it.

Reply:
And here : i need to remove archiveimport allowed : it broke DB :)